install_config/persistent_storage/persistent_storage_manila.adoc

```diff
@@ -0,0 +1,270 @@
+[[persistent_storage_manila]]
+= Persistent Storage Using OpenStack Manila
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+[IMPORTANT]
+====
+Persistent volume Provisionining using OpenStack Manila is a Technology Preview
+feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+{product-title} is capable of provisioning PV using the
+link:https://wiki.openstack.org/wiki/Manila[OpenStack Manila] shared file system service.
+
+It is assumed the OpenStack Manila service has been correctly set up and is
+accessible from the {product-title} cluster. Only the NFS share types can be
+provisioned.
+
+Familiarity with
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs],
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent
+volume claims (PVCs)], and
+link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning] is recommended.
+
+[[installation-setup]]
+== Installation and Setup
+
+The featuere is provided by an external provisioner that needs to be installed and
+configured in the {product-title} cluster.
+
+[[start-controller-and-provisioner]]
+=== Starting the External Provisioner
+
+The external provisioner service is distributed as a container image and can be run in the
+{product-title} cluster as usual. There is also RPM versions for provisioner.
```

Comment:
s/There is also RPM versions/There are also RPM versions available for the provisioner.

Reply:
The RPMs live in one of the RHEL channels. But since we would like the users to use only the containers in productions, I will not mention the RPMs.